### PR TITLE
feat: only wait 60s draining a node during reset

### PR DESCRIPTION
**Pull request by laverya**

### CODE REVIEW

File: cmd/embedded-cluster/reset.go [hunk 1]
```diff
@@ -91,6 +91,7 @@ func (h *hostInfo) drainNode() error {
 		"drain",
 		"--ignore-daemonsets",
 		"--delete-emptydir-data",
+		"--timeout", "60s",
 		h.Hostname,
```
Comment (chris-sanders) [inline cmd/embedded-cluster/reset.go:95]: Shouldn't we also pass --delete-local-data

**Comment by chris-sanders** [comment]
Only other comment I have, from the story have you: "All other commands have been reviewed and confirmed to already have a timeout or one is added".

IE. Do you see any other commands that don't have a way to timeout and proceed in the reset command other than this one?

**Comment by laverya** [comment]
I'll take a look

**Review by chris-sanders — APPROVED** [review]
As long as tests pass this looks good, thanks!

**Pull request merged**